doc/nrf/protocols/wifi/scan_mode/scan_operation.rst

```diff
@@ -136,12 +136,12 @@ The following controls can fine-tune the time taken and power consumed by a scan
        | :c:member:`wifi_scan_params.scan_type`
    * - Active dwell time
      - | Build time:
-       | :kconfig:option:`CONFIG_WIFI_MGMT_SCAN_DWELL_TIME_ACTIVE`
+       | :kconfig:option:`CONFIG_WIFI_SCAN_DWELL_TIME_ACTIVE`
```

Comment:
Can this be removed from here since it is defined in the sample?

Reply:
Done